docs/process/pairing.md

```diff
@@ -0,0 +1,88 @@
+2. **Distraction reduction**. Pairing forces your attention to be immediately accountable to another person. Checking your phone? Bad form.
+3. **Team building / skill development.** Pairing promotes the exchange of ideas and skills. One developer may be more experienced or have a different area of expertise than the other, and pairing allows for continuous, hands-on learning. 
+
+## What
+
+If pairing means working together, what does the work look like?
+
+We have found three axes helpful in answering this question:
+
+* Driver vs. Navigator
+* Scheduled vs. ad hoc
+* Unblocking a problem vs. exploring a problem space
+
+It is typical of pair programming to have two roles: a **driver** (the person with control of the editor) and a **navigator** (the person keeping track of where we are, giving immediate feedback and information to the driver). 
+
+It is good practice to be explicit about declaring which role each person is taking at the start of a pair.
+It is similarly good practice to switch roles during the pair. 
+
+Conversely, it is extremely poor practice to be disengaged during a pair regardless of role.
+This is a trap that is easy to fall into as a navigator and must be consciously avoided.
+
+**Scheduled** pairs are straightforward: you agree to a time in advance.
+**Ad hoc** pairs happen immediately.
+You must actively communicate personal preferences and performance factors when being asked to pair. 
+
+Do not accept invites or requests to pair if you _know_ you are not going to be an active, great communicator.
+Consider asking “is now still a good time to pair?” when you begin.
+The only pair you are obliged to accept is one with the CTO.
+That said, if you find yourself never offering or accepting a pair don’t be surprised to hear feedback about teamwork.
+
+Finally, deciding whether you are **unblocking a problem** vs. **exploring a problem space** can have big impact on the work of a pair.
+Unblocking requires extreme clarity about what exactly the problem is.
+If you bring a problem to your pair you should strongly consider providing details in advance.
+Lack of preparation and clarity of thought signals disengagement from the very start.
+If you post about being blocked by a problem in your standup, consider asking for a pair.
+
+Exploring a problem space, to the contrary, sometimes means learning what exactly the problem is during the pair.
+This type of pairing can be especially useful during the earliest phases of feature development. 
+
+Along all three axes, personal preferences and performance factors should be communicated.
+If you prefer unblocking problems as a navigator in ad hoc pairs, you should make sure to say it. 
+
+## How
+
+Pairing requires active engagement by both people.
+There is no passive role.
+
+Typically, at least when in person, pairing means sharing one computer to do the work together.
+Working on separate machines is considered bad etiquette at best and at worst missing the point entirely. 
```

Comment:
I was surprised to read this. In person, you (@jdjkelly) and I paired on separate machines and I didn't perceive it as bad etiquette. In fact I thought pairing on the same machine may have limited the amount of screen/monitor real estate that we could use and may have hindered our ability to move fast. What do you think?

Reply:
You may not have noticed but I sat away from my laptop, and at times closed it entirely. I didn't use my machine throughout the pair except when I announced that I was doing something. When we needed to use one of my accounts, we actually didn't use my machine but instead screen shared it onto yours. All of that was intentional for the reasons I write here.